f3/granite.go

```diff
@@ -65,22 +67,27 @@ type GMessage struct {
 	// ID of the sender/signer of this message (a miner actor ID).
 	Sender ActorID
 
-	Current SignedMessage
-	// VRF ticket for CONVERGE messages (otherwise empty byte array).
-	Ticket Ticket
+	// Current is the payload that is signed by the signature
+	Current Payload
```

Comment:
We gotta find a better name for this. Payload would be better. Value would be ok, though overloaded (since it also contains a value). Body? Proposal? Vote?